backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ProjectDatabaseHandler.java

```diff
@@ -230,6 +234,14 @@ private boolean duplicateAttachmentExist(Project project) {
         return false;
     }
 
+    private boolean updateProjectAllowed(Project project, User user) {
+        if (project.clearingState != null && project.clearingState.equals(ProjectClearingState.CLOSED)
```

Comment:
minor simplification:
```suggestion
        return ! (project.clearingState != null && project.clearingState.equals(ProjectClearingState.CLOSED) && !PermissionUtils.isUserAtLeast(UserGroup.SW360_ADMIN, user));
```